integration-tests/deployment/ccip/view/chain.go

```diff
@@ -16,11 +16,12 @@ type ChainView struct {
 	TokenAdminRegistry map[string]v1_5.TokenAdminRegistryView `json:"tokenAdminRegistry,omitempty"`
 	CommitStore        map[string]v1_5.CommitStoreView        `json:"commitStore,omitempty"`
 	// v1.6
-	FeeQuoter    map[string]v1_6.FeeQuoterView    `json:"feeQuoter,omitempty"`
-	NonceManager map[string]v1_6.NonceManagerView `json:"nonceManager,omitempty"`
-	RMN          map[string]v1_6.RMNRemoteView    `json:"rmn,omitempty"`
-	OnRamp       map[string]v1_6.OnRampView       `json:"onRamp,omitempty"`
-	OffRamp      map[string]v1_6.OffRampView      `json:"offRamp,omitempty"`
+	FeeQuoter          map[string]v1_6.FeeQuoterView    `json:"feeQuoter,omitempty"`
+	NonceManager       map[string]v1_6.NonceManagerView `json:"nonceManager,omitempty"`
+	RMN                map[string]v1_6.RMNRemoteView    `json:"rmn,omitempty"`
+	OnRamp             map[string]v1_6.OnRampView       `json:"onRamp,omitempty"`
+	OffRamp            map[string]v1_6.OffRampView      `json:"offRamp,omitempty"`
+	CapabilityRegistry map[string]v1_6.CapRegView       `json:"capabilityRegistry,omitempty"`
```

Comment:
The contract is versioned 1.0 though

Reply:
yeah this is a bit of a weird case, its shared/multi-product 1.0 contract which we introduced in 1.6. I think still makes most sense to include in 1.6

Reply:
How about adding a comment? so that we don't end up changing it later